exercises/book-store/tests/book-store.rs

```diff
@@ -0,0 +1,135 @@
+//! Tests for book-store
+//!
+//! Generated by [script][script] using [canonical data][canonical-data]
+//!
+//! [script]: https://github.com/exercism/rust/blob/master/bin/init_exercise.py
+//! [canonical-data]: https://raw.githubusercontent.com/exercism/problem-specifications/master/exercises/book-store/canonical_data.json
+
+extern crate book_store;
+use book_store::*;
+
+/// Process a single test case for the property `total`
+///
+/// All cases for the `total` property are implemented
+/// in terms of this function.
+///
+/// Note that you'll need to both name the expected transform which
+/// the student needs to write, and name the types of the inputs and outputs.
+/// While rustc _may_ be able to handle things properly given a working example,
+/// students will face confusing errors if the `I` and `O` types are not concrete.
```

Comment:
Should it be the case that the student still sees this? They may be confused because I and O are nowhere in sight here.